content/en/examples/pods/pod-with-affinity-anti-affinity.yaml

```diff
@@ -8,10 +8,11 @@ spec:
       requiredDuringSchedulingIgnoredDuringExecution:
         nodeSelectorTerms:
         - matchExpressions:
-          - key: kubernetes.io/os
```

Comment:
Hello @deRemo . Just checking if the `pods/pod-with-affinity-anti-affinity.yaml` file was updated by mistake? The content page (`assign-pod-node.md`) refers to the example file, `pods/pod-with-node-affinity.yaml`?